transcendence-app/src/game/game.controller.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Post,
+  Query,
+  ServiceUnavailableException,
+  UnprocessableEntityException,
+  UseGuards,
+} from '@nestjs/common';
+import { GameService } from './game.service';
+import { TwoFactorAuthenticatedGuard } from '../shared/guards/two-factor-authenticated.guard';
+import {
+  ApiBadRequestResponse,
+  ApiCreatedResponse,
+  ApiForbiddenResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiParam,
+  ApiServiceUnavailableResponse,
+  ApiTags,
+  ApiUnprocessableEntityResponse,
+} from '@nestjs/swagger';
+import { Game } from './infrastructure/db/game.entity';
+import { MAX_ENTRIES_PER_PAGE } from '../shared/constants';
+import { PaginationWithSearchQueryDto } from '../shared/dtos/pagination-with-search.query.dto';
+import { CreateGameDto } from './dto/create-game.dto';
+
+@Controller()
+@UseGuards(TwoFactorAuthenticatedGuard)
+@ApiTags('game')
+@ApiForbiddenResponse({ description: 'Forbidden' })
+export class GameController {
+  constructor(private gameService: GameService) {}
+
+  @Post('game')
+  @ApiCreatedResponse({ description: 'Add a game', type: Game })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiUnprocessableEntityResponse({ description: 'Unprocessable entity' })
+  async addGame(@Body() game: CreateGameDto): Promise<Game> {
+    const savedGame = await this.gameService.addGame(game);
+
+    if (!savedGame) {
+      throw new UnprocessableEntityException();
+    }
+    return savedGame;
+  }
+
+  @Delete('game')
+  @HttpCode(HttpStatus.NO_CONTENT)
+  @ApiParam({ name: 'gameId', type: String })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiNotFoundResponse({ description: 'Not Found' })
+  @ApiServiceUnavailableResponse({ description: 'Service Unavailable' })
+  async removeGame(gameId: string): Promise<void> {
+    return this.gameService.deleteGame(gameId);
+  }
```

Comment:
suggestion: We could limit these endpoints to website owners and moderators because, currently, all the authenticated users can access them.